garage/tf/plotter/plotter.py

```diff
@@ -98,6 +98,9 @@ def _start_worker(self):
 
     def shutdown(self):
         if self.worker_thread.is_alive():
+            while not self.queue.empty():
+                self.queue.get()
+                self.queue.task_done()
```

Comment:
```
Traceback (most recent call last):
  File "examples/tf/trpo_cartpole.py", line 25, in <module>
    algo.train()
  File "/Users/jonathon/Documents/garage/garage/garage/tf/algos/batch_polopt.py", line 146, in train
    self.shutdown_worker()
  File "/Users/jonathon/Documents/garage/garage/garage/tf/algos/batch_polopt.py", line 101, in shutdown_worker
    self.plotter.shutdown()
  File "/Users/jonathon/Documents/garage/garage/garage/tf/plotter/plotter.py", line 103, in shutdown
    self.queue.task_done()
  File "/anaconda2/envs/garage/lib/python3.6/queue.py", line 68, in task_done
    raise ValueError('task_done() called too many times')
ValueError: task_done() called too many times
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/anaconda2/envs/garage/lib/python3.6/queue.py", line 68, in task_done
    raise ValueError('task_done() called too many times')
ValueError: task_done() called too many times
```

Reply:
I fixed this by moving some calls of task_done previously inserted to the right places. Right after the tasks are completed in the worker thread.